Content.Shared/Prying/Systems/PryingSystem.cs

```diff
@@ -101,13 +101,13 @@ private bool CanPry(EntityUid target, EntityUid user, PryingComponent? comp = nu
 
         if (comp != null)
         {
-            canev = new BeforePryEvent(user, comp.PryPowered, comp.Force);
+            canev = new BeforePryEvent(user, comp.PryPowered, comp.Force, false);
         }
         else
         {
             if (!TryComp<PryUnpoweredComponent>(target, out _))
                 return false;
-            canev = new BeforePryEvent(user, false, false);
+            canev = new BeforePryEvent(user, false, false, true);
         }
 
```

Comment:
"CanXYZ" events should be idempotent and not issuing popup messages or the likes tbh. Also having a readonly silent field on the event is really strange, maybe just have the message passed into the event instead for any potential popups is better than have TryPry handle it instead.

Reply:
I made it so systems that check for the event can pass a message to it. Although displaying the message is now done from TryPry, not sure where else to put it as it's where we determine if prying fails and there are multiple possible messages for failure.